e3sm_diags/plot/enso_diags_plot.py

```diff
@@ -41,6 +41,35 @@
 # Use 179.99 as central longitude due to https://github.com/SciTools/cartopy/issues/946
 PROJECTION = ccrs.PlateCarree(central_longitude=179.99)
 
+# Border padding relative to subplot axes for saving individual panels
+# (left, bottom, right, top) in page coordinates
+ENSO_BORDER_PADDING_MAP = (-0.07, -0.025, 0.2, 0.035)
+
+
+def _save_plot_scatter(fig: plt.Figure, parameter: EnsoDiagsParameter):
+    """Save the scatter plot using a simplified approach for single panel plots."""
+    import os
+
+    from e3sm_diags.driver.utils.io import _get_output_dir
+
+    for f in parameter.output_format:
+        f = f.lower().split(".")[-1]
+        fnm = os.path.join(
+            _get_output_dir(parameter),
+            parameter.output_file + "." + f,
+        )
+        plt.savefig(fnm)
+        logger.info(f"Plot saved in: {fnm}")
```

Comment:
This block of code is identical in `_save_plot()`. We can break that part of as its own function and re-use it here, instead of copying the same code.